### PR TITLE
sfs: add a changelog

### PR DESCRIPTION
**Pull request by jecluis**
Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Comment by jecluis** [comment]
let's merge this before building the final container for v0.1.0

**Review by 0xavi0 — APPROVED** [review]
LGTM

File: src/rgw/store/sfs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on
+[Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0]
```
Comment (giubacc) [inline src/rgw/store/sfs/CHANGELOG.md:9]:
```suggestion
## [0.1.0] - 2022-07-14
```

**Review by giubacc — CHANGES_REQUESTED** [review]

**Review by giubacc — APPROVED** [review]
LGTM

**Pull request merged**